src/lib.rs

```diff
@@ -188,8 +137,8 @@ impl CommonMarkCache {
     }
 
     /// Refetch all images
-    pub fn reload_images(&mut self) {
-        self.images.lock().clear();
+    pub fn reload_images(&mut self, ui: &Ui) {
+        ui.ctx().forget_all_images();
```

Comment:
This has a different meaning now. Before it removed only egui_commonmark's images, now it removes _all_ images.  The extra tracking of images required just for this function does not seem worth it to me, so I'm tempted to have the entire method removed. I added this method originally just because I could...